static/frame.js

```diff
@@ -24,6 +24,7 @@ function frame_preload(endFrame) {
 function update_frame(state, val) {
```

Comment:
@allenywang Should the `update_frame` function be merged with the `Canvas.frame` setter?